docs/api/geometries/ConeGeometry.html

```diff
@@ -42,7 +42,7 @@
 
 		<h2>Constructor</h2>
 
-		<h3>[name]([page:Float radius], [page:Float height], [page:Integer radialSegments], [page:Integer heightSegments], [page:Boolean openEnded], [page:Float thetaStart], [page:Float thetaLength])</h3>
+		<h3>[name]([page:Float radius], [page:Float height], [page:Integer radiusSegments], [page:Integer heightSegments], [page:Boolean openEnded], [page:Float thetaStart], [page:Float thetaLength])</h3>
```

Comment:
The name of the parameter is actually `radialSegments` 😉 , see https://github.com/mrdoob/three.js/blob/dev/src/geometries/ConeGeometry.js#L10

And i think it's okay like this